ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/FileObjectPutter.java

```diff
@@ -37,7 +37,6 @@ public FileObjectPutter(final Path root) {
         this.root = root;
     }
 
-    @Override
```

Comment:
This function isn't actually overriding anything - its implementing the interface